code-challenges.js

```diff
@@ -9,44 +9,117 @@
 // Pseudo code:
 
 // Set one:
-const fruit1 = "apple"
-const fruit2 = "banana"
+// const fruit1 = "apple"
+// const fruit2 = "banana"
 
 // Set two:
-const fruit3 = "cherry"
-const fruit4 = "kiwi"
+// const fruit3 = "cherry"
+// const fruit4 = "kiwi"
 
 
-// --------------------1) Create a function that takes in any given number and determines if the number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below. Expected output: "42 is below boiling point", "350 is above boiling point", "212 is at boiling point"
+// --------------------1) Create a function that taßkes in any given number and determines if the number is below boiling point, at boiling point, or above boiling point. Boiling point is 212 degrees Fahrenheit. Use the test variables provided below. Expected output: "42 is below boiling point", "350 is above boiling point", "212 is at boiling point"
 
 // Pseudo code:
 
 const temp1 = 42
 const temp2 = 350
 const temp3 = 212
 
+//------ I CREATED A FUNCTION CALLED boilingPoint... IT TAKES ANY GIVING NUMBER (temp)...
 
-// --------------------2) Create the code that will combine the two arrays and return the length using the variables provided below. Expected output: 10
+const boilingPoint = (temp) => {
+
+//------ I CREATED if STATEMENTS TO DETERMINE IF THE NUMBER (temp) WAS AT BOILING POINT (temp3)... BELOW BOILING POINT (temp1)... AND AFTER BOILING POINT (temp2)... 
+    
+    if(temp === temp3) {
+      return '212 is at boiling point'
+    }
+    else if(temp === temp1) 
+    {
+      return '42 is below boiling point'
+    }
+    else if(temp === temp2) 
+    {
+      return '350 is above boiling point'
+    }
+}
+    console.log(boilingPoint(212))
+
+    console.log(boilingPoint(42))
+
+    console.log(boilingPoint(350))
```

Comment:
I like that you are thinking of conditional statements for this.  Just keep in mind what you are evaluating in your condition.  What is temp vs temp3?